Makefile

```diff
@@ -0,0 +1,90 @@
+#***************************************************************************
+# set options
+#***************************************************************************
+
+export LATEST_RELEASE_REDIRECT = $(shell curl -s https://github.com/curl/curl/releases/latest -w'%{redirect_url}\n' -o /dev/null)
```

Comment:
Not sure I like this. Reproducible builds would dictate that you just have LATEST_RELEASE_VERSION="fixed string", and when you wanted to increment the version that's being used, you update the fixed string and make a commit out of it. Otherwise you're not pinning the external dependency to a version.

Reply:
yes I was being lazy, pinned LATEST_RELEASE_TAG and LATEST_RELEASE_VERSION